lms/static/sass/shared-v2/_components.scss

```diff
@@ -47,3 +47,48 @@
         }
     }
 }
+
+// TODO: search box should be in the Pattern Library
+.page-header-search {
+    display: inline-block;
+
+    .search-form {
+        display: inline-block;
+    }
+
+    .search-input {
+        display: inline-block;
+        position: relative;
+        vertical-align: middle;
+    }
+
+    .search-field {
+        @include padding($baseline/4, $baseline*1.5, $baseline/4, $baseline/2);
+        transition: all $tmg-f2 ease-in-out;
+        border: 1px solid $gray-l4;
+        border-radius: 3px;
+        font-family: inherit;
+    }
+
+    .action-search {
+        @include padding($baseline/5, $baseline/2, $baseline/5, $baseline/2);
+        text-shadow: none;
+        vertical-align: middle;
+    }
+
+    .action-clear {
+        @include right(0);
+        @include margin(0, ($baseline/4), 0, 0);
+        @include padding($baseline/4, $baseline/4, $baseline/4, $baseline/4);
```

Comment:
I thought I had to use the function to get RTL to work. I can see that maybe that isn't necessary when the left and right sides are symmetrical. I thought it would be better to consistently use the function, but then it is harder to use and harder to read.